src/main/java/org/gridsuite/modification/modifications/olg/OlgsModification.java

```diff
@@ -0,0 +1,209 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.modifications.olg;
+
+import com.powsybl.commons.report.ReportNode;
+import com.powsybl.commons.report.TypedValue;
+import com.powsybl.iidm.network.*;
+import org.gridsuite.modification.dto.*;
+
+import java.util.*;
+
+import static org.gridsuite.modification.dto.OperationalLimitsGroupInfos.Applicability.*;
+import static org.gridsuite.modification.dto.OperationalLimitsGroupInfos.Applicability.EQUIPMENT;
+import static org.gridsuite.modification.dto.OperationalLimitsGroupModificationType.DELETE;
+import static org.gridsuite.modification.modifications.AbstractBranchModification.*;
+
+/**
+ * handles the modification of a list of operational limits groups from AbstractBranchModification
+ *
+ * @author Mathieu DEHARBE <mathieu.deharbe at rte-france.com>
+ */
+public class OlgsModification {
```

Comment:
For file title I think the expand name is better, OperationalLimitsGroupsModification